### PR TITLE
First set of examples

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: examples/lusid/portfolios/create-reference-portfolio.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- #################### SUMMARY ############################
+
+-- 1. In this query, we crate a Reference Portfolio in LUSID
```
Comment (JMcDanielFbn) [inline examples/lusid/portfolios/create-reference-portfolio.sql:3]: typo: `create` (other examples of this typo in this MR too)

File: examples/lusid/portfolios/create-reference-portfolio.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+@@portfolioCode1 = select 'uk-equity-model';
+
+-- Define the portfolio data
+@create_portfolio = select 'Reference' as PortfolioType,
+@@portfolioScope as PortfolioScope,
+@@portfolioCode1 as PortfolioCode,
+@@portfolioCode1 as DisplayName,
+'' as Description,
+#2000-01-01# as Created,
+'GBP' as BaseCurrency;
```
Comment (JMcDanielFbn) [inline examples/lusid/portfolios/create-reference-portfolio.sql:18]: This is an example of where we could improve style (via Pretty print or manually)
```
@create_portfolio = 
select 
   'Reference' as PortfolioType,
   @@portfolioScope as PortfolioScope,
   @@portfolioCode1 as PortfolioCode,
   @@portfolioCode1 as DisplayName,
   '' as Description,
   #2000-01-01# as Created,
   'GBP' as BaseCurrency;
```

File: examples/system/error-handling-details-to-file.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+from Lusid.Instrument.SimpleInstrument.Writer
+where ToWrite = @instruments_for_upload;
+
+@writer_status = select count (*) as ResultCount from @writer_results  where WriteErrorDetail is not null;
+
+@target_file_result =
+
+select iif(ResultCount = 0,
+(@@processed_path),
+(@@error_path))
```
Comment (JMcDanielFbn) [inline examples/system/error-handling-details-to-file.sql:25]: Extra parenthesis `(@@x)` -> `@@x`

**Review by rizwansaeed — APPROVED** [review]

File: examples/system/create-view-with-no-params.txt [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+@model_portfolios = use Drive.csv
+--file=/luminesce-temp-system-testing-folder/model_portfolios.csv
+enduse;
+
+select
+    #distinct
+  #select
+{
+    {ModelPortfolioName^ : model_port_name}, -- we add a ^ to indicate this is a main field
```
Comment (JMcDanielFbn) [inline examples/system/create-view-with-no-params.txt:43]: Note sure why but I thought `--` wasn't supported in the middle of these macros

File: examples/system/create-view-with-no-params.txt [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    {Sector: sect},
+    {Region: loc},
+    {Weighting: weighting}
+}
+
+from @model_portfolios
+where #restrict
```
Comment (JMcDanielFbn) [inline examples/system/create-view-with-no-params.txt:50]: using `select_agg` and `restrict_agg` might be better in this example

**Pull request merged**